packages/@aws-cdk-containers/ecs-service-extensions/README.md

```diff
@@ -133,6 +133,51 @@ At this point, all the service resources will be created. This includes the ECS
 Definition, Service, as well as any other attached resources, such as App Mesh Virtual
 Node or an Application Load Balancer.
 
+## Built-in extensions
+
+The following extensions come built-in, and ready to use.
+
+- ### `new AppMeshExtension({ mesh })`
+
+  This extension adds an Envoy Proxy sidecar to your service, and creates an App Mesh virtual node and virtual service to represent your service inside the service mesh. It accepts an AppMesh service mesh as input:
+
+  ```js
+  const mesh = new appmesh.Mesh(stack, 'my-mesh');
+  myService.add(new AppMeshExtension({ mesh }));
+  ```
+
+  If two service both have an `AppMeshExtension` added to them you can connect the two services to each other using the `connectTo()` method like this:
+
+  ```js
+  serviceA.connectTo(serviceB)
+  ```
+
+  The `AppMeshExtension` will respond to this connection request by creating a service mesh route between these two services, and opening their security groups so service A can connect to service B.
+
+- ### `new FireLensExtension()`
```

Comment:
Will you be adding sample code here showing how to invoke all of the other extensions?

Reply:
Yes I'm planning to do another doc pass PR on the end prior to merging the overall feature branch. This is just the framework starting out